tests/test_binary_pointwise_ops.py

```diff
@@ -1844,3 +1845,21 @@ def test_accuracy_logical_xor(shape, dtype):
         res_out = torch.logical_xor(inp1, inp2)
 
     gems_assert_equal(res_out, ref_out)
+
+
+@pytest.mark.polar
+@pytest.mark.parametrize("shape", POINTWISE_SHAPES)
+@pytest.mark.parametrize("dtype", [torch.float32, torch.float64])
```

Comment:
some vendors don't support torch.float64 on their chips. and we don't require them to run tests of torch.float64 successfully. it's okay to check torch.float32 only.